### PR TITLE
feat: UI fix remove whitespaces from messages list

### PR DESCRIPTION
**Pull request by Esther-Lita**
### TL;DR

Added trimStart() to eliminate leading whitespace from convo message preview list. Couldn’t reproduce on my environment. please test on review. 

### What changed?

Modified convo-list-item.tsx to adjust convo message text rendering.

### How to test?

1. Retrieve a conversation with leading whitespace in the first message.
2. Verify that the whitespace is correctly trimmed from the displayed message.

### Why make this change?

Ensures a cleaner UI and prevents layout issues caused by unexpected leading whitespace in conversation messages.

---

## What does this PR do?

Fixes #ENG-119

_If there is not an issue for this, please create one first. This is used to tracking purposes and also helps use understand why this PR exists_

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Chore (refactoring code, technical debt, workflow improvements)
- [ ] Enhancement (small improvements)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### Required

- [ ] Read [Contributing Guide](https://github.com/un/inbox/blob/main/CONTRIBUTING.md)
- [ ] Self-reviewed my own code
- [ ] Tested my code in a local environment
- [ ] Commented on my code in hard-to-understand areas
- [ ] Checked for warnings, there are none
- [ ] Removed all `console.logs`
- [ ] Merged the latest changes from main onto my branch with `git pull origin main`
- [ ] My changes don't cause any responsiveness issues

### Appreciated

- [ ] If a UI change was made: Added a screen recording or screenshots to this PR
- [ ] Updated the UnInbox Docs if changes were necessary

### CODE REVIEW

**Comment by Esther-Lita** [comment]
* **#668** <a href="https://app.graphite.dev/github/pr/un/inbox/668?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @Esther-Lita and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: apps/web/src/app/[orgShortcode]/convo/_components/convo-list-item.tsx [hunk 1]
```diff
@@ -181,7 +181,7 @@ export const ConvoItem = memo(function ConvoItem({
               </div>
 
               <span className="line-clamp-2 overflow-ellipsis whitespace-break-spaces break-words">
-                {convo.entries[0]?.bodyPlainText ?? ''}
+                {convo.entries[0]?.bodyPlainText.trimStart() ?? ''}
```
Comment (BlankParticle) [inline apps/web/src/app/[orgShortcode]/convo/_components/convo-list-item.tsx]:
```suggestion
                {(convo.entries[0]?.bodyPlainText ?? '').trimStart()}
```
trimStart might throw if there is no body

**Review by BlankParticle — APPROVED** [review]

**Review by BlankParticle — CHANGES_REQUESTED** [review]
~

File: apps/web/src/app/[orgShortcode]/convo/_components/convo-list-item.tsx (hunk 1 above)
Comment (Esther-Lita) [reply apps/web/src/app/[orgShortcode]/convo/_components/convo-list-item.tsx]: I have pushed a change a few hours ago to use trim() alone.

Comment (BlankParticle) [reply apps/web/src/app/[orgShortcode]/convo/_components/convo-list-item.tsx]: use it like this `(convo.entries[0]?.bodyPlainText ?? '').trim()`

**Review by BlankParticle — APPROVED** [review]
its fine

**Pull request merged**